suite-native/config/src/supportedNetworks.ts

```diff
@@ -12,37 +11,39 @@ export const mainnetsOrder: NetworkSymbol[] = [
     'ada',
     'bch',
     'xrp',
-    'dash',
     'zec',
-    'btg',
-    'vtc',
-    'nmc',
-    'dgb',
 ];
 
-export const testnetsOrder: NetworkSymbol[] = [
+export const supportedTestnetNetworks: NetworkSymbol[] = [
     'test',
     'regtest',
     'tsep',
     'tgor',
     'thol',
     'tada',
     'txrp',
-    'dsol',
 ];
 
-export const supportedNetworkSymbols = Object.keys(networks) as NetworkSymbol[];
+export const supportedNetworkSymbols = [...supportedMainnetNetworks, ...supportedTestnetNetworks];
 
-export const supportedMainnetSymbols = getMainnets().map(network => network.symbol);
+export const supportedMainnetSymbols = supportedMainnetNetworks;
 
-// These networks are enabled for xpub/address import in portfolio tracker.
-export const importEnabledNetworkSymbols = supportedNetworkSymbols.filter(
-    network => !deprecatedNetworks.includes(network),
-);
+export const sortNetworks = (networks: Network[]) =>
+    networks.sort((a, b) => {
```

Comment:
This will mutate the argument, better use `A.sort` which is not mutating

Reply:
[d322446](https://github.com/trezor/trezor-suite/pull/10241/commits/d3224461c37cbe1e69ccf60b6b83ee37767c1c1b)